pkg/deploy/util/util.go

```diff
@@ -72,6 +72,17 @@ func LabelForDeploymentConfig(config *deployapi.DeploymentConfig) string {
 	return fmt.Sprintf("%s/%s:%d", config.Namespace, config.Name, config.LatestVersion)
 }
 
+// ConfigSelector matches all the deployments of the provided DeploymentConfig
+func ConfigSelector(config *deployapi.DeploymentConfig, list []api.ReplicationController) []api.ReplicationController {
+	matches := []api.ReplicationController{}
+	for _, rc := range list {
+		if rc.Annotations[deployapi.DeploymentConfigAnnotation] == config.Name {
```

Comment:
Sorry, yet another thing I forgot to mention: use `DeploymentConfigNameFor` rather than a direct annotation lookup. The annotation names can/have changed and `DeploymentConfigNameFor` insulates you from those changes.